src/core/ChatSDKErrors.ts

```diff
@@ -47,7 +47,9 @@ enum ChatSDKErrors {
     /** Failure on retrieving conversation details */
     ConversationDetailsRetrievalFailure = "ConversationDetailsRetrievalFailure",
     /** Failure on finding the contact id related to the auth code */
-    AuthContactIdNotFoundFailure = "AuthContactIdNotFoundFailure"
+    AuthContactIdNotFoundFailure = "AuthContactIdNotFoundFailure",
+    /** Failure from getChatToken returning 401 */
+    InvalidAuthentication = "InvalidAuthentication"

```

Comment:
I am assuming that this error is generic enough that it could be reuse in the future right? If it is, we can just mention that it's a `Failure in authentication with the service`

Reply:
updated description